core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -486,6 +486,34 @@ class SparkSubmitSuite
     conf.get("spark.kubernetes.driver.container.image") should be ("bar")
   }
 
+  test("SPARK-35084: includes jars passed in through --packages in k8s client driver mode") {
```

Comment:
For me client driver mode sounds out of the terminology.
What about something like
"SPARK-35084: include jars of the --packages in k8s client mode & driver runs inside a POD"?

Reply:
It is fixed ty!